pipelines/build/openjdk15_pipeline.groovy

```diff
@@ -12,7 +12,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-def javaToBuild = "jdk"
```

Comment:
does jdk15u exist?

Reply:
I'm not sure. I mostly just went off all of the other builds that use `jdk<version>u`. How can I find out?

Reply:
https://github.com/AdoptOpenJDK/openjdk-jdk14u exists but https://github.com/AdoptOpenJDK/openjdk-jdk15u doesn't yet (the upstream mirror doesn't exist yet either)

Reply:
Oh ok I see what you mean. The correct repo to use would be https://github.com/AdoptOpenJDK/openjdk-jdk. In that case, due to the tags, it's probably easier to just rename the binary instead of renaming the jdk version at start. I'll reset this branch and make the changes